package.json

```diff
@@ -17,16 +19,20 @@
   "license": "ISC",
   "dependencies": {
     "@hapi/hapi": "^20.2.2",
+    "@prisma/client": "^3.14.0",
     "@types/dotenv": "^8.2.0",
```

Comment:
Hello, why do you need @types/dotenv when you already have "dotenv" as well?
I also found that @types/[dotenv](https://www.npmjs.com/package/@types/dotenv) is deprecated 😯

Reply:
I guess I have to uninstall that package because it isn't used by the project. jajaja